### DISCUSSION TITLE
S-Bahn Hamburg

### DISCUSSION THREAD

**Discussion by C1710** [marhei/TrainCompany-Data#54] [Ideas]
Theoretisch könnte man hier ein zusätzliches TrainEquipment (und Linien) erstellen für den S-Bahn-Betrieb mit Gleichstrom.
Einziges Problem: Derzeit gibt es keine Möglichkeit, die Fahrzeuge auf ein bestimmtes Equipment zu beschränken.

Theoretisch könnte man das "normale" Netz mit einem Equipment versehen (wäre im Zusammenhang mit #31 sinnvoll).
Alternativ kann die aktuelle Generation (BR 490) aber auch im normalen Bahnnetz mit 15kV fahren?

**Comment by marhei** [comment]
Ja, das Problem, dass man aktuell keine Fahrzeuge für "normale" Strecken sperren kann ist mir bekannt und wird auch angegangen.
